Libraries/Components/WebView/WebView.ios.js

```diff
@@ -166,6 +166,10 @@ var WebView = React.createClass({
     );
   },
 
+  evaluateJavaScript: function(script: string, callback: Function) {
+    return RCTWebViewManager.evaluateJavaScript(this.getWebWiewHandle(), script, callback);
```

Comment:
The error here is in the definition - it is actually defined getWebWiewHandle in the file.